src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -310,6 +325,18 @@ public void Stop()
         }
 
         #region private 
+        private void NotifyError(Exception exception)
+        {
+            // Throw and catch the exception.  This lets it be seen by the debugger
+            // ETW, and other monitoring tools.   However we immediately swallow the
+            // exception.   We may wish in the future to allow users to hook this 
+            // in other useful ways but for now we simply swallow the exceptions.  
+            try
+            {
```

Comment:
How about Debug.Write on the error?
This way it's easier to find out during debugging that something went wrong right in VS.